core/src/main/java/org/sonarsource/sonarlint/core/container/global/ExtensionInstaller.java

```diff
@@ -94,6 +94,16 @@ private static void loadExtensions(ComponentContainer container, PluginInfo plug
     }
   }
 
+  private static boolean onlySonarSourceSensor(PluginInfo pluginInfo, Object extension) {
+    return PluginCacheLoader.isWhitelisted(pluginInfo.getKey()) || isNotSensor(extension);
+  }
+
+  private static boolean isSonarLintSide(Object extension) {
+    return ExtensionUtils.isSonarLintSide(extension)
+      // FIXME Waiting for InputFileFilter to be annotated @SonarLintSide
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Take the required action to fix the issue indicated by this comment. ([squid:S1134](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&branch=feature%2Finputfile_filter&issues=AWEtcf9lr8h_fVHZ4bW4&open=AWEtcf9lr8h_fVHZ4bW4)

[ISSUE_KEY]: AWEtcf9lr8h_fVHZ4bW4